blacklistfilter/evaluator/split_evidence/fields.c

```diff
@@ -0,0 +1,12 @@
+/************* THIS IS AUTOMATICALLY GENERATED FILE, DO NOT EDIT *************/
```

Comment:
fields.c file should not be versioned in git repo, it should be generated automatically by Makefile